cibuildwheel/windows.py

```diff
@@ -154,6 +154,16 @@ def setup_setuptools_cross_compile(
     # set the platform name
     map_plat = {"32": "win32", "64": "win-amd64", "ARM64": "win-arm64"}
     plat_name = map_plat[python_configuration.arch]
+
+    # Set environment variable so that setuptools._distutils.get_platform()
+    # identifies the target, not the host
+    vscmd_arg_tgt_arch = {"32": "x86", "64": "x64", "ARM64": "arm64"}
+    current_tgt_arch = vscmd_arg_tgt_arch[python_configuration.arch]
+    if env.get("VSCMD_ARG_TGT_ARCH", current_tgt_arch) != current_tgt_arch:
```

Comment:
Tiny readability suggestion...
```suggestion
    if "VSCMD_ARG_TGT_ARCH" in env and env["VSCMD_ARG_TGT_ARCH"] != current_tgt_arch:
```

Reply:
This doesn't handle having it set to an empty variable, which apparently one of CI's does. 🙄 We should treat empty variables like unset variables, I think.

```suggestion
    if (env.get("VSCMD_ARG_TGT_ARCH") or current_tgt_arch) != current_tgt_arch:
```